### PR TITLE
add multi-root support to workspace-uri-contribution

### PR DESCRIPTION
**Pull request by elaihau**
Signed-off-by: elaihau <liang.huang@ericsson.com>

### CODE REVIEW

File: packages/workspace/src/browser/workspace-uri-contribution.ts [hunk 1]
```diff
@@ -86,13 +106,11 @@ export class WorkspaceUriLabelProviderContribution extends DefaultUriLabelProvid
      */
     getLongName(element: URI | FileStat): string {
         const uri = this.getUri(element);
```
Comment (akosyakov) [inline packages/workspace/src/browser/workspace-uri-contribution.ts]: it should be:
```ts
const uri = this.getUri(element);
return uri ? this.workspaceService.getWorkspaceRootUri(uri) : super.getLongName(uri)
```

After https://github.com/theia-ide/theia/pull/3613/files#diff-70a982fe47d80b1a0ac76c9c84fd5a4b gets merged.

**Comment by akosyakov** [comment]
actually it was resolved in https://github.com/theia-ide/theia/pull/3538 already

**Comment by elaihau** [comment]
@akosyakov 
> actually it was resolved in #3538 already

I am not sure if they are equivalent.
I will wait your PR to get merged and test.

File: packages/workspace/src/browser/workspace-uri-contribution.ts [hunk 2]
```diff
@@ -86,13 +106,11 @@ export class WorkspaceUriLabelProviderContribution extends DefaultUriLabelProvid
...
-
-        if (this.wsRoot) {
-            const relativeUri = this.wsRoot.relative(uri);
+        if (this.baseUri) {
+            const relativeUri = this.baseUri.relative(uri);
```
Comment (akosyakov) [inline packages/workspace/src/browser/workspace-uri-contribution.ts]: btw it looks bogus, we should workspace root which is the parent, not first random

Comment (elaihau) [reply packages/workspace/src/browser/workspace-uri-contribution.ts]: no it is not the `first random` :) 

the code uses the first root when there is only one root folder in the workspace.

**Comment by elaihau** [comment]
> @elaihau #3538 was merged, can this PR be closed?

Tested the master branch. This one can be closed. Thank you !

**Pull request closed without merging**